v3/src/components/axis/hooks/use-sub-axis.ts

```diff
@@ -177,6 +177,10 @@ export const useSubAxis = ({
       if (!subAxisElt) return
       subAxisSelectionRef.current = select(subAxisElt)
       const sAS = subAxisSelectionRef.current
+      if (sAS.classed('numeric-axis')) {
+        sAS.selectAll('g').remove()
+        sAS.classed('numeric-axis', false)
+      }
```

Comment:
Is there anything comparable that needs to happen when switching from categorical back to numeric?